packages/@jsii/go-runtime/jsii-runtime-go/kernel/callbacks.go

```diff
@@ -1,20 +1,146 @@
 package kernel
```

Comment:
This file was changed to no longer actually expose a user-accessible API... The reason being there is no use for users to call `Callbacks` directly, as this feature is currently unused, and in any case should be hidden from end-user code, and entirely managed by the runtime library.

This now handles "inline callback requests" received as part of the standard request-response flow, and does not actually handle asynchronous methods anymore (this is unused - and would be re-introduced in the future if needed).